renderer/index.js

```diff
@@ -595,6 +597,10 @@ function addTorrentEvents (torrent) {
     var torrentSummary = getTorrentSummary(torrent.infoHash)
     torrentSummary.status = 'seeding'
     updateTorrentProgress()
+    torrent.getFileModtimes(function (err, fileModtimes) {
+      if (err) return onError(err)
+      torrentSummary.fileModtimes = fileModtimes
```

Comment:
Can't test on Mac, LMK if this works for you: https://github.com/feross/webtorrent-desktop/pull/278/commits/682e9275c8176ba57e09909a366a51015eae34e5